index.html

```diff
@@ -55,5 +55,146 @@ <h1> Hey there. I’m Alvin <br>
   <button class="btn arrow"> <i class="fa fa-angle-down"> </i></button>
   </section>
   </section>
+
+  <section class="works_section">
+  <div class="recent_works_title">
+  <h1>My Recent Works</h1>
+  </div>
+  <div class="cards">
```

Comment:
- [ ] Please Follow the[ Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988) and add what is missing in your design.   

- [ ] **Original Design**

![see1](https://user-images.githubusercontent.com/63972670/159130787-a05d25e8-5665-4f8b-87eb-e9fa5c2a7587.JPG)

- [ ] **Your Design**

![see2](https://user-images.githubusercontent.com/63972670/159130825-476d69c2-d1bf-4f6f-a095-dc6d6e042702.JPG)